### ISSUE TITLE
Setup Facebook Marketing source fails

### ISSUE THREAD

**Issue by saitai0802** [airbytehq/airbyte#3991]
## Expected Behavior
After configurated the setting of Facebook Marketing source, it should be able to set up the source sucessfully.

## Current Behavior
It is now fails during the connection tests. It returns
`Error: 2635, (#2635) You are calling a deprecated version of the Ads API. Please update to the latest version: v11.0.`

## Logs
```
2021-06-09 13:20:59 INFO (/tmp/workspace/09cb9090-1f85-4ae5-892a-5649dfbbe57c/0) TemporalAttemptExecution(get):111 - Executing worker wrapper. Airbyte version: 0.24.7-alpha
2021-06-09 13:20:59 INFO (/tmp/workspace/09cb9090-1f85-4ae5-892a-5649dfbbe57c/0) LineGobbler(voidCall):85 - Checking if airbyte/source-facebook-marketing:0.2.7 exists...
2021-06-09 13:20:59 INFO (/tmp/workspace/09cb9090-1f85-4ae5-892a-5649dfbbe57c/0) LineGobbler(voidCall):85 - airbyte/source-facebook-marketing:0.2.7 was found locally.
2021-06-09 13:20:59 INFO (/tmp/workspace/09cb9090-1f85-4ae5-892a-5649dfbbe57c/0) DockerProcessFactory(create):111 - Preparing command: docker run --rm --init -i -v airbyte_workspace:/data -v /tmp/airbyte_local:/local -w /data/09cb9090-1f85-4ae5-892a-5649dfbbe57c/0 --network host airbyte/source-facebook-marketing:0.2.7 check --config source_config.json
2021-06-09 13:21:00 ERROR (/tmp/workspace/09cb9090-1f85-4ae5-892a-5649dfbbe57c/0) DefaultAirbyteStreamFactory(internalLog):108 - Error: 2635, (#2635) You are calling a deprecated version of the Ads API. Please update to the latest version: v11.0.
2021-06-09 13:21:00 ERROR (/tmp/workspace/09cb9090-1f85-4ae5-892a-5649dfbbe57c/0) DefaultAirbyteStreamFactory(internalLog):108 - Check failed
2021-06-09 13:21:00 INFO (/tmp/workspace/09cb9090-1f85-4ae5-892a-5649dfbbe57c/0) TemporalAttemptExecution(get):134 - Stopping cancellation check scheduling...
```

## Steps to Reproduce
1. Click set up the source with source type facebook marketing
2. Fill in all the configurations then click set up source

## Severity of the bug for you
High

## Airbyte Version
0.24.7-alpha

## Connector Version (if applicable)
ource-facebook-marketing:0.2.7

## Additional context
It is a newly created environment that I wanted to try Airbyte features.

**Comment by keu** [comment]
@saitai0802 this being fixed since version 0.2.11, you can update the version of facebook connector in the admin interface.

**Comment by saitai0802** [comment]
Thank you for the update @keu  !
